src/schema/v2/HeroUnit/HeroUnitType.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { InternalIDFields } from "../object_identification"
+import { ResolverContext } from "types/graphql"
+import Image, { normalizeImageData } from "../image"
+
+interface HeroUnitTypeProps {
```

Comment:
#minor but props seems like a weird naming, these are basically the type of the Gravity API response so I might call it that for clarity